scripts/ci/run-integration-tests

```diff
@@ -5,49 +5,183 @@
 # replace it with a test runner in the future, please see it more as a proof of
 # concept
 
-set -e
```

Comment:
We don't want to exit on error? I understand this for the test execution part, but if something else fails we want to error out, right?

Reply:
For me the plus is 

- I can not forget to do a `|| exit 1` at the end of each statement
- Writing `|| exit 1` at the end of each statement feels repetitive
- If some command fails in an unexpected way (e.g. rsync not being installed on some machine) the script can go rouge without me noticing at first, because it just runs over it. Best case I need to search for the error line and fix it, worst case it screws up my system because some assumptions fail and weird stuff happens
- I almost never want to have a command in my script just fail randomly (except for cypress here). For me, it is the same as writing `try{}catch(e){}` around every statement of my JS code.

Reply:
ok, added 👍